src/components/panel/panel.js

```diff
@@ -1663,6 +1795,40 @@ MdPanelRef.prototype._done = function(callback, self) {
 };
 
 
+/**
+ * Adds a panel to a group if the panel does not already exist within the group.
```

Comment:
The doc string mention "if the panel does not already exist within the group", but I don't see anything that checks that the panel isn't part of the group.